### DISCUSSION TITLE
Cannot open module file 'groundwatermmfmod.mod'

### DISCUSSION THREAD

**Discussion by nicetynio** [NCAR/hrldas#93] [Q&A] [answered]
I encountered an error message saying 'Fatal Error: Cannot open module file 'groundwatermmfmod.mod' for reading at (1): No such file or directory.' After checking the 'src' folder in the 'noahmp' directory, I found that the 'NoahmpGroundwaterInitMod.F90' file, which used to be there, was deleted during the compilation process. How could this file have been deleted? I have attached my 'user_build_options' file for reference.

#=============================================================================================
#  Options for Linux with gfortran (not fully tested)
#=============================================================================================

 COMPILERF90    =       /usr/bin/gfortran
 FREESOURCE     =       -ffree-form  -ffree-line-length-none
 F90FLAGS       =       -g -fconvert=big-endian -fbounds-check -fno-range-check -fno-underscoring
 MODFLAG        =       
 LDFLAGS        =       
 CPP            =       /usr/bin/cpp
 CPPFLAGS       =       -P -traditional -D_GFORTRAN_
 LIBS           =       
 LIBJASPER      =       
 INCJASPER      =        
 NETCDFMOD      =       -I/usr/include
 NETCDFLIB      =       -L/usr/lib -lnetcdf -lnetcdff
 BZIP2          =       
 BZIP2_LIB      =       
 BZIP2_INCLUDE  =       
 RM             =       rm -f
 CC             =       /usr/bin/gcc

**Comment by cenlinhe** [comment, marked as answer]
This issue has been resolved in the latest HRLDAS v5.0 by updating the Makefile. Please try again to see if your problem disappears.
